### PR TITLE
add zap metadata

### PR DESCRIPTION
**Pull request by damonmcc**
related to https://github.com/NYCPlanning/data-engineering/issues/800

### CODE REVIEW

File: products/zap/bbls/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  \ (LUCATS) and covers all projects that have been \u2018Noticed\u2019 (that they\
+  \ will appear before the City Planning Commission in 30 or more days for Certification\
+  \ as complete for the ULURP process to begin) or Filed (a CEQR or Land Use application\
+  \ has been formally submitted to the Department for review) through completion (approval,\
+  \ disapproval, withdrawal, or termination). For more information on the land use\
+  \ and environmental review application process see:\r\nhttps://www1.nyc.gov/site/planning/applicants/applicant-portal/application-process.page\r\
+  \n\r\nYou can explore this data in the Zoning Application Portal: https://zap.planning.nyc.gov/projects\r\
+  \n\r\nTo report data errors or for questions, email <a href=\"mailto:zap_feedback_dl@planning.nyc.gov\"\
+  >zap_feedback_dl@planning.nyc.gov </a>."
+description: "The Department of City Planning (DCP) processes land use applications\
```
Comment (alexrichey) [inline products/zap/bbls/metadata.yml:18]: @damonmcc Let's format the long lines with nice yaml style. [this](https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines) might be helpful. 

TLDR: remove carriage-returns and newlines, `\u2019` chars, and backslashes. Also, we should just remove the summary from our models. I can't remember quite what it was for, but it's not used, and now just an annoyance.

**Review by alexrichey — CHANGES_REQUESTED** [review]
Looks good! Let's clean up the descriptions while we're here, then 👍

**Comment by damonmcc** [comment]
@alexrichey
> Looks good! Let's clean up the descriptions while we're here, then 👍

see [fdcf5bd](https://github.com/NYCPlanning/product-metadata/pull/3/commits/fdcf5bdce5a060424fbc4e01aec2278b590064ce)

**Review by alexrichey — APPROVED** [review]
Cool - FYI, can add the xlsx's back
